source/style-guide/style/punctuation.txt

```diff
@@ -347,7 +349,9 @@ Periods
 ~~~~~~~
 
 Use the following guidelines for periods. For basic period usage, see a
-grammar reference, `section 6.12, Periods <https://www.chicagomanualofstyle.org/book/ed17/part2/ch06/psec012.html>`__ in the *Chicago Manual of Style*.
+grammar reference, `section 6.12, Periods
+<https://www.chicagomanualofstyle.org/book/ed17/part2/ch06/psec012.html>`__
+in the *Chicago Manual of Style*.
```

Comment:
Apologies if it wasn't part of the JIRA ticket task to fix all the instances of CMOS, but it looks like theres no link in the "Hyphens with prefixes" section which I spotted when searching for this link.

Reply:
Done. 

The other links are paywalled. Opened https://jira.mongodb.org/browse/DOCS-15855 to replace them.